test/osb_and_plugin_test/provision_timeout_test/provision_timeout_suite_test.go

```diff
@@ -0,0 +1,256 @@
+	username, password := test.RegisterBrokerPlatformCredentials(SMWithBasicPlatform, emptyCatalogBrokerID)
+	brokerPlatformCredentialsIDMap[emptyCatalogBrokerID] = brokerPlatformCredentials{
+		username: username,
+		password: password,
+	}
+
+	simpleBrokerCatalogID, _, brokerServerWithSimpleCatalog = ctx.RegisterBrokerWithCatalog(common.SBCatalog(simpleCatalog)).GetBrokerAsParams()
+	smUrlToSimpleBrokerCatalogBroker = brokerServerWithSimpleCatalog.URL() + "/v1/osb/" + simpleBrokerCatalogID
+	common.CreateVisibilitiesForAllBrokerPlans(ctx.SMWithOAuth, simpleBrokerCatalogID)
+	username, password = test.RegisterBrokerPlatformCredentials(SMWithBasicPlatform, simpleBrokerCatalogID)
+	brokerPlatformCredentialsIDMap[simpleBrokerCatalogID] = brokerPlatformCredentials{
+		username: username,
+		password: password,
+	}
+
+	plan0 := common.GenerateTestPlanWithID(plan0CatalogID)
+	service0 := common.GenerateTestServiceWithPlansWithID(service0CatalogID, plan0)
+	catalog := common.NewEmptySBCatalog()
+	catalog.AddService(service0)
+
+	stoppedBrokerID, _, stoppedBrokerServer = ctx.RegisterBrokerWithCatalog(catalog).GetBrokerAsParams()
+	common.CreateVisibilitiesForAllBrokerPlans(ctx.SMWithOAuth, stoppedBrokerID)
+	stoppedBrokerServer.Close()
+	smUrlToStoppedBroker = stoppedBrokerServer.URL() + "/v1/osb/" + stoppedBrokerID
+	username, password = test.RegisterBrokerPlatformCredentials(SMWithBasicPlatform, stoppedBrokerID)
+	brokerPlatformCredentialsIDMap[stoppedBrokerID] = brokerPlatformCredentials{
+		username: username,
+		password: password,
+	}
+
+	plan1 := common.GenerateTestPlanWithID(plan1CatalogID)
+	plan2 := common.GenerateTestPlanWithID(plan2CatalogID)
+	plan3 := common.GenerateTestPlanWithID(plan3CatalogID)
+
+	service1 := common.GenerateTestServiceWithPlansWithID(service1CatalogID, plan1, plan2, plan3)
+	catalog = common.NewEmptySBCatalog()
+	catalog.AddService(service1)
+
+	var brokerObject common.Object
+
+	utils = ctx.RegisterBrokerWithCatalog(catalog)
+	brokerID = utils.Broker.ID
+	brokerObject = utils.Broker.JSON
+	brokerServer = utils.Broker.BrokerServer
+	utils.BrokerWithTLS = ctx.RegisterBrokerWithRandomCatalogAndTLS(ctx.SMWithOAuth).BrokerWithTLS
+
+	plans := ctx.SMWithOAuth.ListWithQuery(web.ServicePlansURL, "fieldQuery="+fmt.Sprintf("catalog_id in ('%s','%s')", plan1CatalogID, plan2CatalogID)).Iter()
+	for _, p := range plans {
+		common.RegisterVisibilityForPlanAndPlatform(ctx.SMWithOAuth, p.Object().Value("id").String().Raw(), ctx.TestPlatform.ID)
+	}
+	smBrokerURL = ctx.Servers[common.SMServer].URL() + "/v1/osb/" + brokerID
+	brokerName = brokerObject["name"].(string)
+
+	username, password = test.RegisterBrokerPlatformCredentials(SMWithBasicPlatform, brokerID)
+	brokerPlatformCredentialsIDMap[brokerID] = brokerPlatformCredentials{
+		username: username,
+		password: password,
+	}
+
+	provisionRequestBody = buildRequestBody(service1CatalogID, plan1CatalogID)
+})
+
```

Comment:
why do you need so much brokers and plans?
just create one with single plan for test